source/features/reactions-avatars.tsx

```diff
@@ -118,5 +133,5 @@ void features.add(import.meta.url, {
 	include: [
 		pageDetect.isDiscussion,
 	],
-	init: onetime(discussionInit),
+	init: discussionInit,
```

Comment:
I need some extra eyes on this file. helpppp review

Reply:
@fregante Should I move those changes to a separate PR?